freeipa/src/main/java/com/sequenceiq/freeipa/sync/StackStatusCheckerJob.java

```diff
@@ -105,23 +105,28 @@ public void syncAStack(Stack stack) {
                     if (!checkableInstances.isEmpty()) {
                         SyncResult syncResult = freeipaChecker.getStatus(stack, checkableInstances);
                         if (DetailedStackStatus.AVAILABLE == syncResult.getStatus()) {
-                            for (Map.Entry<InstanceMetaData, DetailedStackStatus> entry : syncResult.getInstanceStatusMap().entrySet()) {
```

Comment:
I think the old way of looping outside of the `updateInstanceStatus()` was cleaner.

Reply:
Thanks! I changed back to the old way of looping outside now and kept the method signature.